src/nagini_contracts/contracts.py

```diff
@@ -26,7 +26,8 @@
 CONTRACT_FUNCS = ['Assume', 'Assert', 'Old', 'Result', 'Implies', 'Forall',
                   'Exists', 'Low', 'Acc', 'Rd', 'Fold', 'Unfold', 'Unfolding',
                   'Previous', 'RaisedException', 'Sequence', 'ToSeq', 'MaySet',
-                  'MayCreate',]
+                  'MayCreate', 'getMethod', 'getArg', 'getOld', 'arg', 'MayJoin',
+                  'MayStart',]
```

Comment:
The MaySet, MayCreate and MayStart functions don't really have anything to do with obligations, the first two are about dynamic fields, the last about thread starting. MayJoin corresponds to the Joinable function from the obligations paper, but it is not an obligation itself. So it could make sense to move MayJoin, but IMO not the others. What do you think?

Reply:
Oh also, I decided to rename MayJoin to Joinable, since that makes it sound less like a permission and that's the name used in the paper.